src/main/java/opensearchSDK/netty4/Netty4MessageChannelHandler.java

```diff
@@ -30,7 +30,7 @@
  * GitHub history for details.
  */
 
-package transportservice.netty4;
+package opensearchSDK.netty4;
```

Comment:
Let this be opensearchSDK.transport.xyz ? 
This is a direct package from https://github.com/opensearch-project/OpenSearch/blob/main/modules/transport-netty4/src/main/java/org/opensearch/transport/netty4/Netty4MessageChannelHandler.java#L33

Lets keep it consistent and whenever we close https://github.com/opensearch-project/OpenSearch/issues/3118
We can remove this piece of code easily.